tests/test_linear_transformation_scene.py

```diff
@@ -5,22 +5,22 @@
 
 def test_ghost_vectors_len_and_types():
     manager = Manager(LinearTransformationScene)
-    scene = manager.scene
+    scene: LinearTransformationScene = manager.scene
```

Comment:
It's interesting that we need this annotation - I thought it should be able to infer this already?

Reply:
Yeah. Honestly, I don't know why this happens.